text/0000-capabilities-based-access-modifers.md

```diff
@@ -0,0 +1,72 @@
+- Feature Name: Capability Based Access Modifiers
+- Start Date: 16-10-2016
+- RFC PR: (leave this empty)
+- Pony Issue: (leave this empty)
+
+# Summary
+
+While capabilities is a language feature, Pony does not use this for the benefit 
+of access modifiers.
+
+# Motivation
+
+Capabilities based access modifiers will support more fine grain protection including:
+
+ - fine grain access modifiers
+ - type state / phantom types
```

Comment:
Since Pony doesn't provide phantom types, it might help to include a little more information about them, either here or in the "Detailed Design" section.